src/js/contexts/AnnounceContext/AnnounceContext.js

```diff
@@ -13,11 +14,11 @@ const createAnnouncer = () => {
 };
 
 export const AnnounceContext = React.createContext(
-  (message, mode = 'polite', timeout = 500) => {
+  (message, mode = 'polite', timeout = 500, id = 'grommet-announcer') => {
```

Comment:
I don't think we should be setting a new default value. can we keep the default behavior as it is now?

Reply:
Letting default behavior as it is now won't fix the issue if people don't explicit add one id.
The addition of default value doesn't cause any side and negative effect. Although, it may prevent further issues.
Please let me know if you still wanting remove it.

Reply:
I think it's OK to add this as the default.